src/readStream.js

```diff
@@ -63,9 +92,16 @@ function createReadStream(wofConfig, types, wofAdminRecords) {
   const wofRoot = wofConfig.datapath;
   const metaFilePaths = getMetaFilePaths(wofRoot, types);
 
-  return createMetaRecordStream(metaFilePaths, types)
-  .pipe(isNotNullIslandRelated.create())
-  .pipe(loadJSON.create(wofRoot, wofConfig.missingFilesAreFatal))
+  // Select correct stream between meta and SQLite based on config and do specialized stuff
+  const stream = wofConfig.sqlite ?
```

Comment:
This ternary checks that `wofConfig.sqlite` is truthy, so a value such as "false" or "no" would also trigger this behaviour.

It would be more robust if the check was `wofConfig.sqlite === true`, or checked against a specific subset of truthy values.

Reply:
Okay, I changed this with `wofConfig.sqlite === true`, Joi will do the job for truthy values